recipe/meta.yaml

```
@@ -6,36 +6,56 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pytest-subprocess-{{ version }}.tar.gz
-  sha256: d7693b96f588f39b84c7b2b5c04287459246dfae6be1dd4098937a728ad4fbe3
+  url: https://github.com/aklajnert/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 80172ac95439cd4a237c5ee7f343227a7fd21fb61284bfa0de4b7802502d88dc
 
 build:
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
   number: 0
+  skip: true  # [py<37]
```

```suggestion
  skip: true  # [py<36]
```
https://inspector.pypi.io/project/pytest-subprocess/1.5.0/packages/44/96/de22cf4a31bf5f21e6b74dc57e3628e3bc78847acf7a1752bdb1e36dfaf1/pytest_subprocess-1.5.0-py3-none-any.whl/pytest_subprocess-1.5.0.dist-info/METADATA#line.29

---

aw.. ehehe thanks!